app/(onboard)/assessment.tsx

```
@@ -1,271 +1,360 @@
+              placeholderTextColor="#9CA3AF"
+              multiline
+              textAlignVertical="top"
+              value={currentAnswer || ""}
+              onChangeText={(text) => handleAnswer(currentQuestion.id, text)}
+            />
+          </View>
+        )
+
       case QuestionType.CHECKBOX:
-        return <CheckboxQuestion {...questionProps} />;
+        const selectedItems = Array.isArray(currentAnswer) ? currentAnswer : []
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)